server/src/client/client-manager.ts

```diff
@@ -120,9 +129,7 @@ export const clientManager = new class {
             try {
                 id = tokenService.decode(request.token).id;
                 token = request.token;
-            } catch {
-                // ignore
-            }
+            } catch { }
```

Comment:
Would recommend keeping the comment. It shows that the decision was made with the intention of ignoring it. That, or turn this into a separate function that returns null if error

Reply:
Ahhhh i understand the decision now and that does make more sense